certbot/docs/contributing.rst

```diff
@@ -483,7 +483,7 @@ Note that instead of just importing ``typing``, due to packaging issues, in Cert
 
```

Comment:
The line just above this talks about `magic_typing` and should be updated too I think. I wrote https://github.com/certbot/certbot/pull/8709 before seeing this change and I think you should feel free to use whatever combination of our PRs you want to resolve this.